cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -100,7 +102,13 @@ func NewCMServer() *CMServer {
 		NodeMemory:              resource.MustParse("3Gi"),
 		SyncNodeList:            true,
 		ClusterName:             "kubernetes",
+		PluginMgr:               controller.NewPluginMgr(),
 	}
+	s.PluginMgr.InitFlags(ProbeControllerPlugins())
+	t := 5 * time.Second
+	s.PluginMgr.SyncPeriods["pvclaimbinder-sync-period"] = &t
```

Comment:
is there any reason you need to use all lower case here? pvclaimbinder is very mysterious unless you already know which four letters are supposed to be capitalized.